segfault on linux when pager should be used

I am testing an using of pager, and I got a segfault

```
export PAGER=pspg
export USQL_PAGER=pspg

[pavel@localhost ~]$ ./usql --version
usql 0.9.0

pg:localhost=> \pset pager on
Pager usage is on.
pg:localhost=> select * from pg_class;
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x850719]

goroutine 1 [running]:
github.com/xo/tblfmt.(*TableEncoder).tableHeight(0xc000782900, 0xc000f00000, 0x18d, 0x200, 0x0)
	/home/ken/src/go/pkg/mod/github.com/xo/tblfmt@v0.6.3/encode.go:455 +0x79
github.com/xo/tblfmt.(*TableEncoder).Encode(0xc000782900, 0x252da20, 0xc000010018, 0xc0007c4240, 0xc000aa1e30)
	/home/ken/src/go/pkg/mod/github.com/xo/tblfmt@v0.6.3/encode.go:183 +0x825
github.com/xo/tblfmt.(*TableEncoder).EncodeAll(0xc000782900, 0x252da20, 0xc000010018, 0x7, 0x8)
	/home/ken/src/go/pkg/mod/github.com/xo/tblfmt@v0.6.3/encode.go:254 +0x45
github.com/xo/tblfmt.EncodeAll(0x252da20, 0xc000010018, 0x25622b0, 0xc000e6a580, 0xc000aa1ad0, 0x0, 0x0)
	/home/ken/src/go/pkg/mod/github.com/xo/tblfmt@v0.6.3/tblfmt.go:43 +0xa2
github.com/xo/usql/handler.(*Handler).query(0xc0005860b0, 0x2557cd0, 0xc000be32c0, 0x252da20, 0xc000010018, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/ken/src/go/src/github.com/xo/usql/handler/handler.go:952 +0x3da
github.com/xo/usql/handler.(*Handler).execOnly(...)
	/home/ken/src/go/src/github.com/xo/usql/handler/handler.go:845
github.com/xo/usql/handler.(*Handler).Execute(0xc0005860b0, 0x2557cd0, 0xc000be32c0, 0x252da20, 0xc000010018, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/ken/src/go/src/github.com/xo/usql/handler/handler.go:414 +0x2d5
github.com/xo/usql/handler.(*Handler).Run(0xc0005860b0, 0x0, 0x0)
	/home/ken/src/go/src/github.com/xo/usql/handler/handler.go:378 +0x525
main.run(0xc00022cd90, 0xc0002d5540, 0x0, 0x0)
	/home/ken/src/go/src/github.com/xo/usql/main.go:147 +0x4f6
main.main()
	/home/ken/src/go/src/github.com/xo/usql/main.go:49 +0x189
```

--- a/go.mod
+++ b/go.mod
@@ -82,7 +82,7 @@ require (
 	github.com/vertica/vertica-sql-go v1.1.1
 	github.com/vmihailenco/msgpack/v5 v5.3.1 // indirect
 	github.com/xo/dburl v0.6.1
-	github.com/xo/tblfmt v0.7.1
+	github.com/xo/tblfmt v0.7.2
 	github.com/xo/terminfo v0.0.0-20210125001918-ca9a967f8778
 	github.com/zaf/temp v0.0.0-20170209143821-94e385923345
 	github.com/ziutek/mymysql v1.5.4

